spec/lib/declare_schema/migration_generator_spec.rb

```diff
@@ -30,7 +30,9 @@
     end
   end
   let(:datetime_precision) do
-    if defined?(Mysql2)
+    if ActiveSupport::VERSION::MAJOR >= 7
```

Comment:
📓 Rails 7+ now sets datetime precision to 6 instead of 0 for mysql or blank for sqlite.